core/src/main/scala/io/finch/Input.scala

```diff
@@ -15,12 +15,17 @@ import shapeless.Witness
  * - Finagle's [[Request]] needed for evaluating (e.g., `body`, `param`)
  * - Finch's route (represented as `Seq[String]`) needed for matching (e.g., `path`)
  */
-final case class Input(request: Request, route: Seq[String]) {
+final case class Input(request: Request, route: Seq[String], method: Method) {
```

Comment:
Can we update the doc for this class to include `method`? Maybe something like:
```
 * An input for [[Endpoint]] that glues three individual pieces together:
 *
 * - Finagle's [[Request]] needed for evaluating (e.g., `body`, `param`)
 * - Finch's route (represented as `Seq[String]`) needed for matching (e.g., `path`)
 * - Finagle's [[Method]] needed for matching the the request's HTTP Verb
```

Reply:
Sure, thanks for pointing out